packages/api-media-karma/jest.config.js

```diff
@@ -5,15 +5,17 @@ module.exports = {
   },
   globals: {
     'ts-jest': {
-      tsConfig: 'src/tsconfig.json'
+      tsconfig: 'src/tsconfig.json'
```

Comment:
Should be tsConfig, no? https://huafu.github.io/ts-jest/user/config/tsConfig

Reply:
I remember that after updating packages it shouted that `tsConfig` is deprecated and I should use `tsconfig` instead